### PR TITLE
[In review] Introduce v1.19 and update devpreview schemas

### PR DESCRIPTION
**Pull request by chu-tianshu**
This change adds v1.19 app schema version.

### CODE REVIEW

File: teams/vDevPreview/MicrosoftTeams.schema.json [hunk 1]
```diff
@@ -1296,6 +1312,40 @@
             "required": [
                 "declarativeCopilots"
```
Comment (juna-gupta) [inline teams/vDevPreview/MicrosoftTeams.schema.json:1313]: declarativeAgents

File: teams/vDevPreview/MicrosoftTeams.schema.json [hunk 2]
```diff
@@ -2894,24 +2944,119 @@
...
+                "name": {
+                    "type": "string",
+                    "enum": [
+                        "bots",
+                        "staticTabs",
+                        "composeExtensions",
+                        "configurableTabs"
+                    ]
+                },
+                "id": {
```
Comment (juna-gupta) [inline teams/vDevPreview/MicrosoftTeams.schema.json:2978]: need to add id property to configurableTabs and composeExtensions

**Review by juna-gupta — CHANGES_REQUESTED** [review]
Please also validate sample RT manifest against the schema changes.

File: teams/vDevPreview/MicrosoftTeams.schema.json (hunk 2 above)
Comment (chu-tianshu) [reply teams/vDevPreview/MicrosoftTeams.schema.json:2988]: Added

File: teams/vDevPreview/MicrosoftTeams.schema.json (hunk 1 above)
Comment (chu-tianshu) [reply teams/vDevPreview/MicrosoftTeams.schema.json]: Fixed

**Pull request merged**